lib/node/NodeSourcePlugin.js

```diff
@@ -24,6 +25,11 @@ NodeSourcePlugin.prototype.apply = function(compiler) {
 			return require.resolve("node-libs-browser/mock/empty");
 		} else return module;
 	}
+
+	function getStringOfRelativePath(context, pathToModule) {
+		return JSON.stringify(path.relative(context, pathToModule).replace(/\\/g, '/'))
```

Comment:
prefix with `./` if:

* it's not already prefixed with `../`
* it's not an absolute path i. e. returned by `path.relative("C:\\a", "D:\\b")`

Here it's already done this way: https://github.com/webpack/webpack/blob/9c5c6df64f7c53f014be058fa18afde03fe6b77f/lib/NodeStuffPlugin.js#L114-L117

Maybe we could merge these code fragments into a common module.